server/src/main/java/org/apache/iotdb/db/utils/AuthUtils.java

```diff
@@ -188,10 +188,22 @@ public static boolean validatePassword(String originPassword, String encryptPass
    *
    * @param pathA sub-path
    * @param pathB path
-   * @return True if pathA == pathB, or pathA is an extension of pathB, e.g. pathA = "root.a.b.c"
-   *     and pathB = "root.a"
+   * @return True if pathA == pathB, or pathA is an extension of pathB, or pathA is contained in the
+   *     semantics of "*" and "**" in pathB, e.g. pathA = "root.a.b.c" and pathB = "root.a",
+   *     "root.a.b.*" or "root.a.**"
    */
   public static boolean pathBelongsTo(String pathA, String pathB) {
+    if (pathB.endsWith(".*")) {
+      if (pathA.endsWith("`")) {
+        pathA = pathA.substring(0, pathA.lastIndexOf(IoTDBConstant.PATH_SEPARATOR + "`"));
+      } else {
+        pathA = pathA.substring(0, pathA.lastIndexOf(IoTDBConstant.PATH_SEPARATOR));
+      }
```

Comment:
OK, I got it. It seems you should process single quote '' and double quote "" as well.

Reply:
The PR #5696 make it only support back quote as the wrapper of the layer name in path, so we don't need consider single quote and double quote.